cas-server-documentation/installation/OAuth-OpenId-Authentication.md

```diff
@@ -20,6 +20,13 @@ Support is enabled by including the following dependency in the Maven WAR overla
 </dependency>
 {% endhighlight %}
 
+and defining the appropriate keys for JWT signing and encryption:
+
+{% highlight properties %}
+cas.oauth.jwt.signingSecret=
```

Comment:
I think we should provide a bit of explanation as to how these secrets are generated, using which tool, what length/agorithm is required, etc.